apps/src/utils.js

```diff
@@ -1,6 +1,7 @@
-/* global define */
+/* global $, define */
 
 var xml = require('./xml');
+var sanitizeHtml = require('sanitize-html');
```

Comment:
You'll need to also make sure this ends up in package.json (`npm install --save-dev sanitize-html` is the easiest way to accomplish that).